R/reactives.R

```diff
@@ -544,14 +553,17 @@ Observer <- R6Class(
 
     initialize = function(observerFunc, label, suspended = FALSE, priority = 0,
                           domain = getDefaultReactiveDomain(),
-                          autoDestroy = TRUE) {
+                          autoDestroy = TRUE, ..stacktraceon = TRUE) {
```

Comment:
I want it to really stick out at the call site, because you'll be scanning the code for matching `..stacktraceoff..`/`..stacktraceon..` pairs. When you use these arguments then that's the substitute for `..stacktraceon`.